back/validators/rangeSupervision.js

```diff
@@ -99,6 +107,7 @@ module.exports = {
     fields.scheduled_range_supervision_id(body, 'exists'),
     fields.range_supervisor(body, 'exists'),
     fields.notice(body, 'optional'),
+    //fields.supervisor(, 'optional'),
```

Comment:
Avoid holy code artifacts. These pieces of code are always dubious in nature whether these should be kept here or not for some peculiar reason. Remove this line.

Reply:
done